client/js/jsonedit.js

```diff
@@ -533,6 +533,36 @@ const jeCommands = [
       }
       jeUpdateMulti();
     }
+  },
+  {
+    id: 'je_selectSortBy',
+    name: 'sortBy',
+    context: '^button ↦ clickRoutine ↦ [0-9]+ ↦ \\(SELECT\\) ↦ ',
```

Comment:
This makes it only work in button clickRoutines. It can be used in any routine and even subroutines. There should be plenty of examples for those.

Reply:
> This makes it only work in button clickRoutines. It can be used in any routine and even subroutines. There should be plenty of examples for those.

It looks as though simply changing `clickRoutine` to `.*Routine` should fix this?

Reply:
`button` needs to go as well. Just `context: 'Routine ↦ [0-9]+ ↦ \\(SELECT\\) ↦ ',` might work although I vaguely remember that it has to match from the beginning because longer context matches are shown at the top. So probably `context: '^.*Routine ↦ [0-9]+ ↦ \\(SELECT\\) ↦ ',`.

Reply:
Yeah, this will require some thought. That change won't work because I need to figure out which `Routine` it is. I need code similar to that in `jeAddRoutineOperationCommands`, which I don't understand and now need to.

Reply:
I think this commit solves the problem.